test/CMakeLists.txt

```diff
@@ -267,10 +267,12 @@ add_library(tiledb_test_support_lib EXCLUDE_FROM_ALL
   $<TARGET_OBJECTS:TILEDB_CORE_OBJECTS>
   ${TILEDB_TEST_SUPPORT_SOURCES}
 )
+target_link_libraries(tiledb_test_support_lib INTERFACE Catch2::Catch2WithMain)
 
 target_include_directories(tiledb_test_support_lib
```

Comment:
This statement shouldn't be necessary. Include directories should come through the Catch dependency.

Reply:
removed.